docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: Database Access with Redis
+description: How to configure Teleport Database Access with Redis.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Redis is available starting from Teleport `9.0`.
+</Details>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Configure mutual TLS authentication between Teleport and your Redis database.
+- Connect to Redis database through Teleport.
+
+## Prerequisites
+
+- Teleport version `9.0` or newer.
+- Redis database (standalone or cluster) version `6.0` or newer.
+- `redis-cli` installed and added to your system's `PATH` environment variable.
+
+<Admonition type="note" title="Note">
+  Redis `7.0` and RESP3 (REdis Serialization Protocol) are currently not supported.
+</Admonition>
+
+## Step 1/5. Set up Teleport Auth and Proxy
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/5. Create Redis users.
+
+Redis user must be protected by a strong password. We recommend using a password generator built-in in Redis.
+To generate password connect to a Redis instance and call:
+
+```
+ACL GENPASS
+```
+
+or use OpenSSL if you have it installed on your machine
```

Comment:
When do we recommend using OpenSSL vs. `ACL GENPASS`?

Reply:
So this I think is the tricky one. Redis documentation recommends using `ACL GENPASS` to generate password. The problem with this is that you need a Redis running to run that command, so it's not the best advice when you're reading a guide on how to setup a Redis cluster and you're at the beginning, but I wanted to mention it somehow that Redis has such option (I don't mind removing it if you think it's confusing). The best idea that I have is to use OpenSSL to generate password, as I'd assume that now everyone knows how to create a "good password".

@r0mant @smallinsky Any objections?

Reply:
no, I think that mentioning `ACL GENPASS` if fine.